tests/phpunit/Compat/WC_Stripe_Subscriptions_Helper_Test.php

```diff
@@ -174,4 +174,124 @@ public function provide_test_build_subscriptions_detached_messages() {
 			],
 		];
 	}
+
+	/**
+	 * Tests for `is_subscription_payment_method_detached`.
```

Comment:
To be more specific, might it be worth using `@see` for this reference?
```suggestion
	 * Tests for {@see WC_Stripe_Subscriptions_Helper::is_subscription_payment_method_detached()}.
```

Reply:
I followed the existing standards. But that works too 👍